.gitignore

```diff
@@ -1,4 +1,12 @@
-node_modules
-.env
-text.txt
-/node_modules
\ No newline at end of file
+# Node build artifacts
+node_modules
+npm-debug.log
+
+# Local development
+*.env
+*.dev

```

Comment:
This will never work.
Because what you do is :all_files_with_extension_of.env or .dev

What you want to achieve is all files that start with .env
Better to use:

```
.env*
.dev*
```